lib/tasks/assets.rake

```diff
@@ -0,0 +1,10 @@
+namespace :assets do
+  task :precompile => 'assets:env'
+
+  desc 'Load dependencies for asset precompilation'
+  task :env do
+    unless ENV['LOAD_ASSETS']
+      raise "Please set the LOAD_ASSETS environment variable to true if you want to precompile assets!"
```

Comment:
Technically it's only required in production (though it's unlikely anyone's going to try to precompile in another environment).

Maybe also worth a comment clarifying that we can't just set it here on demand because it's too late.